src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -1619,6 +1620,33 @@ namespace winrt::TerminalApp::implementation
         args.Handled(true);
     }
 
+    void TerminalPage::_HandleHandleUri(const IInspectable& /*sender*/,
+                                        const ActionEventArgs& args)
+    {
+        if (const auto& uriArgs{ args.ActionArgs().try_as<HandleUriArgs>() })
+        {
+            const auto uriString{ uriArgs.Uri() };
+            if (!uriString.empty())
+            {
+                Windows::Foundation::Uri uri{ uriString };

```

Comment:
technically this will throw if the URI string is wrong. `wt handle-uri foobarbaz` will maybe make terminal blow up

Reply:
You may want to fix this in a followup!